vita3k/gui/src/settings_dialog.cpp

```diff
@@ -155,6 +155,15 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
     ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
     if (ImGui::BeginTabItem("GPU")) {
         ImGui::PopStyleColor();
+#ifdef USE_VULKAN
+        ImGui::Combo("Backend Render (Reboot for apply)", reinterpret_cast<int *>(&host.backend_renderer), "OpenGL\0Vulkan\0");
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Select your preferred backend render.");
+        if (static_cast<int>(host.backend_renderer) == 0 && host.cfg.backend_renderer != "OpenGL")
```

Comment:
This would be better as switch cases I think.

```cpp
switch (backend_renderer) {
case renderer::Backend::OpenGL:
	...
	break;
case renderer::Backend::Vulkan:
	...
	break;
}
```

Reply:
yes, i know, just i am not fan using case juste for 2 value ^^

Reply:
We will probably have more renderers in the future...

Reply:
hmm yes, hmm exept dx12 no idea what ^^